src/main/java/decodes/tsdb/ComputationApp.java

```diff
@@ -785,13 +785,15 @@ private void checkTimedCompList(long now, Calendar timedCompCal)
 			+ "(select cmp.computation_id, cmp.date_time_loaded "
 				+ "from cp_comp_property cprop, cp_computation cmp "
 				+ "where cprop.computation_id = cmp.computation_id "
-				+ "and lower(prop_name) = 'timedcompinterval'"
+				+ "and lower(prop_name) = 'timedcompinterval' "
+				+ "and cmp.enabled = 'Y' "
 				+ "and cmp.loading_application_id = " + getAppId() + ") q1"
```

Comment:
This could use the new handler/bind var treatment but I'll take care of that later. I vague recall some issues with "truth" values between the various implementations.

Reply:
the text2bool helper looks to accept true, yes, on for reading booleans? These are a one character field in HDB, Y or N by convention. Are the other implementations different here?

Reply:
I need to review. CWMS itself uses 'T' and 'F' but I think all the OpenDCS usages use 'Y' and 'N', I would like to make the whole thing more robust in general. Just not really sure how to do it yet.